src/main/java/uk/gov/hmcts/divorce/document/DocumentUtil.java

```diff
@@ -182,17 +199,64 @@ private static Map<String, List<DocumentType>> documentsApplicableForConfidentia
                 NOTICE_OF_PROCEEDINGS_APP_1,
                 GENERAL_LETTER,
                 AOS_RESPONSE_LETTER,
+                CONDITIONAL_ORDER_REMINDER,
                 CERTIFICATE_OF_ENTITLEMENT_COVER_LETTER_APP1,
                 CONDITIONAL_ORDER_GRANTED_COVERSHEET_APP_1,
-                FINAL_ORDER_GRANTED_COVER_LETTER_APP_1
+                FINAL_ORDER_GRANTED_COVER_LETTER_APP_1,
+                FINAL_ORDER_CAN_APPLY_APP1
             ),
             APPLICANT2, Lists.newArrayList(
                 NOTICE_OF_PROCEEDINGS_APP_2,
                 GENERAL_LETTER,
                 CERTIFICATE_OF_ENTITLEMENT_COVER_LETTER_APP2,
                 CONDITIONAL_ORDER_GRANTED_COVERSHEET_APP_2,
-                FINAL_ORDER_GRANTED_COVER_LETTER_APP_2
+                FINAL_ORDER_GRANTED_COVER_LETTER_APP_2,
+                FINAL_ORDER_CAN_APPLY_APP2
+            )
+        );
+    }
+
+    public static Map<DocumentType, ConfidentialDocumentsReceived> getDocumentTypeToConfidentialTypeMapping() {
+        Map<DocumentType, ConfidentialDocumentsReceived> map = new HashMap<>(
+            Map.of(
```

Comment:
Do you need to do `new HashMap(Map.of(...))` can't it just be `Map.of(...)`

Reply:
Map.of creates an immutable map